### ISSUE TITLE
Home pages: What we mean by disability section - Heading and body text shouldn't be side by side

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#762]
The text for what we mean by disability should be placed below the heading rather than on its right. This change would make navigation at high zoom level easier.

**Comment by cherylhjli** [comment]
@sepidehshahi I center-aligned it, does this help?
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A3399

Did users also have issues with the "how does this work" section being on the right side?

**Comment by sepidehshahi** [comment]
Yes @cherylhjli that's great! thanks

**Comment by sepidehshahi** [comment]
They didn't have issue with how does it work section as there was an image on the left and they naturally navigated to the right to find the rest of the image and they saw the text.

**Comment by greatislander** [comment]
Resolved in #1254.
